CHANGELOG.md

```diff
@@ -14,6 +14,12 @@
 
   Started by [#335](https://github.com/Plutonomicon/plutarch/pull/335)
 
+- Added module `Plutarch.FFI` for interoperability with PlutusTx.
+
+- Added module `Plutarch.Show` with the `PShow` type class.
```

Comment:
A similar entry (along with the link to the PR) was already added to the top of the CHANGELOG. You can merge them.